### PR TITLE
Target follower node

### PR DESCRIPTION
**Pull request by abrandemuehl**
@ananmay3 I'm making a pull request for this so I can review it

### CODE REVIEW

**Comment by abrandemuehl** [comment]
@shubhankar12 You'll have to do the review on this one since I opened it

File: src/motion/basic_target_follower.cpp [hunk 1]
```diff
@@ -11,8 +11,7 @@ constexpr float kAlt = 0.01;
 // Forward velocity of the sub
 constexpr float kSpeed = 0.1;
```
Comment (shubhankar12) [inline src/motion/basic_target_follower.cpp]: Read this from launch file. We want to decide the speed of the submarine when we launch the whole code.

File: src/motion/basic_target_follower_node.cpp [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#include <ros/ros.h>
+#include <string>
+
+int main(int argc, char **argv) {
+  ros::init(argc, argv, "targetfollower");
+  std::string target_topic;
+  if (!ros::param::get("target_topic", target_topic)) {
+    throw ros::Exception("Must specify target_topic parameter");
+  }
+  ros::NodeHandle n;
```
Comment (shubhankar12) [inline src/motion/basic_target_follower_node.cpp]: Change the nodeHandle variable name from `n` to `nh`. 
That seems to be the convention.

File: src/motion/basic_target_follower_node.cpp [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#include "basic_target_follower.h"
+#include <ros/ros.h>
+#include <string>
+
+int main(int argc, char **argv) {
```
Comment (shubhankar12) [inline src/motion/basic_target_follower_node.cpp]: Add some comments. What is this node doing? 
A line or 2 is good. Mention what are you getting from the subscriber and what will you be publishing.

**Comment by abrandemuehl** [comment]
@shubhankar12 Fixed up the requests.

**Pull request merged**